### DISCUSSION TITLE
Week 13 Final Assignment

### DISCUSSION THREAD

**Discussion by gabebott** [USU-ITLS-x265-23-summer/class-repo#241] [General]
This is a simple memory game that includes elements of different types of code. I feel that this project pushed me as a beginning coder, and helped me to understand concepts that we have been working on throughout the semester. I am eager to hear feedback on this game. I enjoyed making it, but it gave me plenty of headaches along the way. Below are my feelings on the levels of breadth and depth that I went into while creating this game.

Intermediate level HTML: I feel that my project did not call for a lot of HTML work, but the for me as a beginner I still felt it to be a
good challenge for me.

Basic level CSS: The CSS was fairly simple cosmetic work on colors and sizes for the different elements of the game.

Advanced level JS: This is where the most headaches for me occured. I spent many hours researching functions and watching/reading different professionals practices when building similar projects. This research ultimately helped me to learn a lot more and reach a level in my code where I am pleased with the final result.

Intermediate-Advanced level Accessibility: I feel that this game is very accessible for people of all ages and that the design of it creates 
an engaging product for all. There is nothing extremely advanced in the design, but I think it does a good job of appealing to a large audience.

Thank you in advance for all of your thoughts, and I am looking forward to hearing from all of you!

-Gabe Bott

Repo
https://github.com/gabebott/Memory-Blitz-Game

Final Project
https://gabebott.github.io/Memory-Blitz-Game/

**Comment by kassyoveson** [comment]
Great job Gabe! I like how your game worked. I can only imagine the headache of getting everything working properly within the code. The end result is fantastic. You should be proud.

**Comment by coopsand** [comment]
I love that you created a game! I haven't really seen anyone do that yet so that was a really cool idea. Great work.

**Comment by bst3ck** [comment]
Well done. Looks great and is actually a nice fun little game. Bravo.

**Comment by jakegros** [comment]
Fantastic, gabebott!  This was fun and a great challenge.  To be honest, I struggled a bit because of the colors being so close and bright.  Obviously you did a wonderful job and put in a lot of hard work.  One thing I would do if you kept going is possibly add a design or number or the color in text on the cards.  I couldn't make some matches due to being colorblind.  However, you definitely crushed this project!!!

**Comment by audreyhki** [comment]
Great job Gabe! I refreshed and played the game multiple times and was impressed that every time, the color cards were in a new/different location.

**Comment by Skye404** [comment]
Great job! It's fun to see everyone's final projects. Some of the colors are a little similar, so that makes it more challenging. I like it.

**Comment by beto0890** [comment]
I love your game! It is really fun!
